rasa/core/processor.py

```diff
@@ -670,13 +670,12 @@ async def _run_action(
             tracker.update(events[0])
             return self.should_predict_another_action(action.name())
         except Exception as e:
-            logger.error(
-                f"Encountered an exception while running action '{action.name()}'. "
+            logger.exception(
+                f"Encountered an exception while running action '{action.name()}'. {e}"
```

Comment:
```suggestion
            logger.exception(
                f"Encountered an exception while running action '{action.name()}'."
```

Because it is using `logger.exception`, it does not need the exception instance in the log message.